### PR TITLE
Issue 24/split schema definitions

### PR DESCRIPTION
**Pull request by justemoilouise**
**Related issue:** https://github.com/dscnitrourkela/project-avocado-web/issues/24

**Description**
- Split schema definitions into types, queries and mutations
- Update `schema.js` to reference definitions from split files

### CODE REVIEW

File: server/schema/schema.js [hunk 1]
```diff
@@ -1,416 +1,7 @@
...
-          return docs;
-        });
-      },
-    },
-  },
-});
+const query = require('./queries');
+const mutation = require('./mutations');
 
 module.exports = new GraphQLSchema({
```
Comment (riteshsp2000) [inline server/schema/schema.js:4]: GraphQLSchema is undefined here. Hence build is failing.

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

**Comment by DesignrKnight** [comment]
@riteshsp2000 
You will review, isn't it?

**Review by riteshsp2000 — APPROVED** [review]

**Comment by riteshsp2000** [comment]
@all-contributors please add @justemoilouise  for code

**Comment by riteshsp2000** [comment]
@justemoilouise Hey. There's been a slight issue while resolving the conflicts. Could you send a pr again?

**Comment by DesignrKnight** [comment]
### Hacktoberfest Thank You from NIT Rourkela community
It was a super fun experience in Hacktoberfest with you. We hope you also had fun and amazing learning. We look forward to you in our upcoming sessions and events as part of the community. 
You can join our Slack community at http://bit.ly/NITRDevs

**As a token of our gratitude, we have set up digital badges of recognition for our contributors from Hacktoberfest. 
Do claim yours at https://live.dscnitrourkela.org/**

Thanks and Cheers!
Stay Safe

**Pull request merged**